tests/conftest.py

```diff
@@ -142,7 +142,7 @@ def create(indexer=INDEXER_TVDBV2, indexerid=0, lang='', quality=Quality.UNKNOWN
 def create_tvepisode(monkeypatch):
     def create(show, season, episode, filepath='', **kwargs):
```

Comment:
As long as everything is working adding more commits to change vars used in methods should be fine since they don't really matter what they're called as long as they're called the same thing through out the method.